### PR TITLE
chore: Added sort-keys and eslint-plugin-import

### PR DESCRIPTION
**Pull request by KarmineSH**
GitHub Issue or Internal Use Azure Devops Work Item ID: 
[Product Backlog Item 306633](https://dev.azure.com/nventive/Practice%20committees/_workitems/edit/306633): Propose changes to the default eslint prettier config

## Proposed Changes

 - [ ] Bug fix
 - [ ] Feature
 - [x] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## What is the current behavior?
We do not have a rule to sort properties and imports

## What is the new behavior?
Properties and imports must be sorted alphabetically according to the rules defined in eslint config

## Checklist

Please check that your PR fulfills the following requirements:

- [ ] Documentation has been added/updated.

### CODE REVIEW

File: frontend/src/app/components/errorHelperText/ErrorHelperText.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
+import "./error-helper-text.scss";
```
Comment (MounirAbdousNventive) [inline frontend/src/app/components/errorHelperText/ErrorHelperText.tsx:1]: C'est une erreur?

File: frontend/package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "zustand": "^4.5.4"
   },
   "devDependencies": {
+    "@types/babel__generator": "^7.6.8",
```
Comment (MounirAbdousNventive) [inline frontend/package.json:39]: Pourquoi ajouter ca?

File: frontend/src/app/containers/debugBanner/DebugBanner.tsx [hunk 3]
```diff
@@ -29,11 +29,11 @@ export default function DebugBanner() {
     <div className={"debug-banner__container"}>
       <div
         className={classNames("debug-banner", {
-          "debug-banner__local": __ENV__ === "local",
           "debug-banner__dev": __ENV__ === "dev",
+          "debug-banner__local": __ENV__ === "local",
```
Comment (MounirAbdousNventive) [inline frontend/src/app/containers/debugBanner/DebugBanner.tsx:33]: c'est dommage de changer l'ordre ici.

File: frontend/src/app/forms/auth/loginForm/LoginForm.tsx [hunk 4]
```diff
@@ -28,8 +28,8 @@ export default function LoginForm({ setIsLoading }: ILoginForm) {
   const navigate = useNavigate();
   const { setUser } = useUserStore();
   const [loginForm, setLoginForm] = useState<ILogin>({
-    username: "",
```
Comment (MounirAbdousNventive) [inline frontend/src/app/forms/auth/loginForm/LoginForm.tsx:31]: J'aime mieu quand les variables sont dans l'ordre du form.

File: frontend/src/app/forms/auth/loginForm/loginForm.schema.ts [hunk 5]
```diff
@@ -1,10 +1,10 @@
 import { object, string } from "yup";
 
 const loginFormSchema = object({
-  username: string().label("login__username").required("validations__required"),
```
Comment (MounirAbdousNventive) [inline frontend/src/app/forms/auth/loginForm/loginForm.schema.ts:4]: L'ordre du form me semblerais plus logique ici aussi

File: frontend/src/app/shared/theme.tsx [hunk 6]
```diff
@@ -6,11 +6,11 @@ const parseMedia = (media: string) => parseInt(media.replace("px", ""));
 const theme = createTheme({
   breakpoints: {
     values: {
-      xs: parseMedia(style["media-xs"]),
-      sm: parseMedia(style["media-sm"]),
-      md: parseMedia(style["media-md"]),
       lg: parseMedia(style["media-lg"]),
+      md: parseMedia(style["media-md"]),
```
Comment (MounirAbdousNventive) [inline frontend/src/app/shared/theme.tsx:10]: Trier par grandeur serait plus logique.

Comment (ChuckLeon) [reply frontend/src/app/shared/theme.tsx:10]: +1
est-ce que c'est eslint qui le force ici?

**Pull request closed without merging**